src/main/java/brokeculator/expense/ExpenseManager.java

```diff
@@ -21,9 +21,13 @@ public void delete(int index) {
         expenses.remove(index);
     }
 
-    public double summariseExpenses() {
+    public double summariseExpenses(int beginIndex, int endIndex) {
         double total = 0;
-        for (Expense expense : expenses) {
+        if (endIndex == -1 || endIndex >= expenses.size()) {
+            endIndex = expenses.size() - 1;
```

Comment:
This is a little nit picky but what if we did `endIndex = expenses.size()` so on line 29 we can do `ArrayList<Expense> expensesToList = new ArrayList<>(expenses.subList(beginIndex, endIndex));`

Reply:
We do `ArrayList<Expense> expensesToList = new ArrayList<>(expenses.subList(beginIndex, endIndex + 1));` on line 29 because we want endIndex to be inclusive, so I think we should keep this implementation